iati/iati/urls.py

```diff
@@ -23,6 +25,10 @@
     url(r'^documents/', include(wagtaildocs_urls)),
 ]
 
+urlpatterns += [
+    url(r'^(104|105|201|202|203)/', reference_redirect)
```

Comment:
On reflection, I would add `101|102|103` to this, given I've just checked the outcome of those URLs

- http://iatistandard.org/101
- http://iatistandard.org/102
- http://iatistandard.org/103

Reply:
:+1: